charts/fission-all/templates/_helpers.tpl

```diff
@@ -31,6 +31,13 @@ canary:
   prometheusSvc: {{ .Values.prometheus.serviceEndpoint | default "" | quote }}
   {{- end }}
   {{- printf "\n" -}}
+auth:
+  enabled: {{ .Values.authentication.enabled | default false }}
+  {{- if .Values.authentication.enabled }}
+  authUrlPath: {{ .Values.authentication.authUrlPath | default "/auth/login" | quote}}
+  jwtExpiryTime: {{ .Values.authentication.jwtExpiryTime | default 120 }}
```

Comment:
I dont see any limitation of this config as of now. But in future if we wish to support different auth methods we need to make sure auth config is flexible for that. Please see from your perspective also we need any changes here from that perspective.